qemu/tests/chardev_acpi.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    acpi description of serial and parallel ports incorrect
+    with -chardev/-device:
+    1) Boot guest A with isa-serial with tty chardev backend
+    2) Enter into guest A
+        udevadm info --query path --name /dev/ttyS0
+        !! --attribute-walk | grep looking
+    3) Boot guest B with -serial /dev/ttyS0
+    4) repeat step 2 for guest B
+    5) Check if the result are same for A and B.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    output = []
+    guest_cmd = params['guest_cmd']
+    for x in range(2):
+        vm = env.get_vm(params["main_vm"])
+        if x >= 1:
+            params['serials'] = ''
+            params['extra_params'] = ' -serial /dev/ttyS0'
+            params['start_vm'] = 'yes'
+        vm.create(params=params)
+        vm.verify_alive()
+        session = vm.wait_for_login()
+        vm_output1 = session.cmd_status_output(guest_cmd)
+        # !! cannot be used because of echo $?
+        vm_output2 = session.cmd_status_output("{} --attribute-walk | grep looking".format(guest_cmd))
+        output.append([vm_output1, vm_output2])
+    assert output.count(output[0]) == len(output)
```

Comment:
Please compare the content.
`output[0]` is 2 absolutely since it run two command for 1 vm.
`len(output)` is 2 absolutely since we test 2 vm.
They are different thing. This compare is meaningless.

Reply:
Hi is it count for output[0] in output i.e if output[1] == output[0] then it is 2 else 1 and the len(output) is 2 always. So it is not meaningless

Reply:
@michiboo ah~  yes, you are right, it is my mistake, thanks!